README.md

```diff
@@ -81,6 +81,14 @@ change signature parameters like the algorithm of the signature.
 
 A string which will be used as single key if `keys` is not provided.
 
+##### encryptionKey
```

Comment:
Just like the signing keys have, the encryption keys also need a method of rotation.